pkg/utils/config.go

```diff
@@ -50,6 +50,8 @@ type CrawlerConfig struct {
 
 	PubSubProjectID string `split_words:"true" desc:"Sets the Google Cloud Project ID name"`
 	PubSubTopicName string `split_words:"true" desc:"Sets the Google Cloud PubSub Topic name"`
+
+	VersionNumber string `split_words:"true" desc:"Sets the version for table"`
```

Comment:
Thoughts on making this a required config?  We should come up with a codified best practice for how we increment the versions.

Reply:
I was thinking it would be annoying to be required bc then we'd always have to pull up the version when we run it... but I guess since it'll be defined in the kubernetes .yaml file it will be better that way since we'll know which version is currently running.

Reply:
oh you are right, let's just leave it.

Reply:
Kept it as not required but probably best practice to run it with the version specified anyway